code/controllers/subsystem/pai.dm

```diff
@@ -197,18 +197,18 @@ var/datum/subsystem/pai/SSpai
 				question(O.client)
 
 /datum/subsystem/pai/proc/question(client/C)
-	spawn(0)
-		if(!C)
-			return
-		asked.Add(C.key)
-		asked[C.key] = world.time
-		var/response = tgalert(C, "Someone is requesting a pAI personality. Would you like to play as a personal AI?", "pAI Request", "Yes", "No", "Never for this round", StealFocus=0, Timeout=askDelay)
-		if(!C)
-			return		//handle logouts that happen whilst the alert is waiting for a response.
-		if(response == "Yes")
-			recruitWindow(C.mob)
-		else if (response == "Never for this round")
-			asked[C.key] = INFINITY
+	set waitfor = 0
+	if(!C)
+		return
+	asked.Add(C.key)
+	asked[C.key] = world.time
+	var/response = tgalert(C, "Someone is requesting a pAI personality. Would you like to play as a personal AI?", "pAI Request", "Yes", "No", "Never for this round", StealFocus=0, Timeout=askDelay)
+	if(!C)
+		return		//handle logouts that happen whilst the alert is waiting for a response.
+	if(response == "Yes")
+		recruitWindow(C.mob)
+	else if (response == "Never for this round")
+		C.prefs.be_special -= ROLE_PAI
```

Comment:
I was thinking of character saves when i made this, which are only saved if you hit the save button in setup character.